backend/src/Core/FurnitureShop.Core.Contracts/Web/Products/CreateProduct.cs

```diff
@@ -1,13 +1,14 @@
 using System;
 using LeanCode.CQRS;
 using LeanCode.CQRS.Security;
+using FurnitureShop.Core.Contracts.Dtos;
 
-namespace FurnitureShop.Core.Contracts.Mobile.Products
+namespace FurnitureShop.Core.Contracts.Web.Products
 {
     [AuthorizeWhenHasAnyOf(Auth.Roles.Admin)]
     public class CreateProduct : IRemoteCommand
     {
-        public ProductDetailsDTO ProductDetails { get; set; }
+        public ProductWithDetailsDTOBase ProductDetails { get; set; }
```

Comment:
Mb it's better to delete ProductWithDetailsDTOBase and use normalized data like single fields, but if you prefer this way, it's ok